components/channel_layout/channel_identifier_router.jsx

```diff
@@ -2,223 +2,122 @@
 // See License.txt for license information.
 
 import React from 'react';
-import {joinChannel} from 'mattermost-redux/actions/channels';
-import {getUser, getUserByUsername, getUserByEmail} from 'mattermost-redux/actions/users';
-import {getTeamByName} from 'mattermost-redux/selectors/entities/teams';
+import $ from 'jquery';
```

Comment:
please replace with pattern used here https://github.com/mattermost/mattermost-webapp/blob/master/components/permalink_view/permalink_view.jsx#L37

Reply:
This is WIP. Please dont review it yet!. It might be waste of time as it is subject for change. Not to worry it will not have jquery API's though.

Reply:
sorry for reviewing too early :) thanks for acknowledging the ban on adding new jquery code to project 🙅